lib/jsonwp-proxy/proxy.js

```diff
@@ -26,15 +26,15 @@ class JWProxy {
   }
 
   endpointRequiresSessionId (endpoint) {
-    return !_.includes(['/session', '/sessions', '/status'], endpoint);
+    return !_.includes(['/session', '/sessions', '/status', '/homescreen'], endpoint);
```

Comment:
I think people will only use this once an Appium session has already been started.